### DISCUSSION TITLE
Random loadout generator

### DISCUSSION THREAD

**Discussion by drackugon238** [janstehno/cotw-companion#22] [Ideas]
I’m just wondering I think it would be something cool but I would understand if it’s not something for this app

**Comment by janstehno** [comment]
It would be interesting indeed. I will think about it as it would be a complex system.

**Comment by janstehno** [comment]
Idea #69 expands this one to the point where this idea becomes redundant and thus will be closed. Please direct further suggestions and questions [here](/janstehno/cotw-companion/discussions/69).
